code/modules/food_and_drinks/food/snacks_vend.dm

```diff
@@ -13,6 +13,32 @@
 	tastes = list("candy" = 1)
 	foodtype = JUNKFOOD | SUGAR
 
+/obj/item/reagent_containers/food/snacks/candy/bronx
+	name = "South Bronx Paradise bar"
+	desc = "Lose weight, guaranteed! Wait, the package actually says 'South Bronx Parasite'..."
```

Comment:
if you're doing the whole "change after one examine" thing, don't mention the parasite part in the desc